AtDigitization/AtDigiLinkDef.h

```diff
@@ -13,5 +13,6 @@
 #pragma link C++ class AtSimulatedLine + ;
 #pragma link C++ class AtTrigger + ;
 #pragma link C++ class AtTriggerTask + ;
+#pragma link C++ class AtVectorResponse + ;
```

Comment:
This never has to be written to disk, so should be `#pragma link C++ class AtVectorResponse - !;`